lib/components/featured_section.dart

```diff
@@ -27,7 +25,10 @@ class FeaturedSection extends StatelessWidget {
     return LayoutBuilder(builder: (context, constraints) {
       const marginX = 2.0;
       final viewportFraction = (constraints.maxWidth - (32 - 2 * marginX)) / max(1, constraints.maxWidth);
-      final sectionItems = data.items.items;
+      final sectionItems = data.items.items.where((e) {
+        final episode = e.item.asOrNull<Fragment$ItemSectionItem$item$$Episode>();
+        return episode == null ? true : !isComingSoon(episode.publishDate);
```

Comment:
```suggestion
        return episode == null || !locked && !isComingSoon(episode.publishDate);
```

Reply:
I guess this update will not be needed after merging this https://github.com/bcc-code/brunstadtv-app/pull/80

Reply:
No because this is more about removing "unclickable" items from featureds

Reply:
So "coming soon" was wrong, sorry, I should have said "locked"

Reply:
Right